kernel/configs/README.md

```diff
@@ -0,0 +1,28 @@
+## How to use defconfig files
+
+defconfig files must be copied in the kernel source code directory at `arch/x86/configs/`
```

Comment:
Capitalize Defconfig and I suggest the following rewrite (lines 3-4): Defconfig files must be copied in the kernel source code directory at `arch/x86/configs/`. Then the `.config` file is generated using the `make` command.